<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addScientistDialog</class>
 <widget class="QDialog" name="addScientistDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>276</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="scientist_gender_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Gender:</string>
   </property>
  </widget>
  <widget class="QLabel" name="scientist_yod_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>190</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Year of Death:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_error_sci_name">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="add_new_scientist_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#550000;&quot;&gt;Add a new scientist&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_error_sci_yob">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>165</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_scientist_yod">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>190</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_add_newScientist_button">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_error_sci_yod">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_scientist_name">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_scientist_yob">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="scientist_yob_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Year of Birth:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_newScientist_button">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLabel" name="scientist_name_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_female">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Female</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_male">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>100</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Male</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_isAlive">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Deceased</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>input_scientist_name</tabstop>
  <tabstop>input_scientist_yob</tabstop>
  <tabstop>input_scientist_yod</tabstop>
  <tabstop>add_newScientist_button</tabstop>
  <tabstop>cancel_add_newScientist_button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
